packages/ckeditor5-dev-tests/bin/check-dependencies.js

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
+
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const fs = require( 'fs' );
+const path = require( 'path' );
+const glob = require( 'glob' );
+
+const cwd = process.cwd();
+const packageJson = require( path.join( cwd, 'package.json' ) );
+
+const currentPackageName = packageJson.name;
+const dependencies = Object.keys( packageJson.dependencies || {} );
+const devDependencies = Object.keys( packageJson.devDependencies || {} );
+
+const missingDependencies = new Set();
+const missingDevDependencies = new Set();
+const invalidFiles = new Set();
+
+const globPattern = path.join( cwd, '@(src|tests)/**/*.js' );
+
+for ( const filePath of glob.sync( globPattern ) ) {
+	fs.readFileSync( filePath, 'utf-8' )
+		.split( '\n' )
+		.forEach( line => {
+			// Find required package name.
+			let packageName = /^import .* from '(@ckeditor\/[^/]+)\/.*/.exec( line );
```

Comment:
This is not a package name here. It's a match for import. If you'd use `const` you wouldn't make this mistake ;)

Reply:
Also, use `[^ ]+` instead of `.*`.

Reply:
It will fail on imports a few modules (`import { getData, setData } from ...`).

Reply:
Right. I've completely forgotten about this case.

Reply:
So at least make it `.+` which is more correct :D

Besides, I've just thought about multi-line imports:

```js
import {
	stringify as stringifyModel,
	setData as setModelData,
	getData as getModelData
} from '@ckeditor/ckeditor5-engine/src/dev-utils/model';
```

Which means that you can't split the lines like that.

This should work though:

```js
/^import[^;]+from '(@ckeditor\/[^/]+)[^']+';/mg/
```

E.g.:

```js
// fs.readFileSync( 'packages/ckeditor5-clipboard/tests/clipboard.js' ).toString().match( /^import[^;]+from '(@ckeditor\/[^/]+)[^']+';/mg );

[ 'import VirtualTestEditor from \'@ckeditor/ckeditor5-core/tests/_utils/virtualtesteditor\';',
  'import Paragraph from \'@ckeditor/ckeditor5-paragraph/src/paragraph\';',
  'import {\n\tstringify as stringifyView,\n\tparse as parseView\n} from \'@ckeditor/ckeditor5-engine/src/dev-utils/view\';',
  'import {\n\tstringify as stringifyModel,\n\tsetData as setModelData,\n\tgetData as getModelData\n} from \'@ckeditor/ckeditor5-engine/src/dev-utils/model\';',
  'import ViewDocumentFragment from \'@ckeditor/ckeditor5-engine/src/view/documentfragment\';',
  'import ViewText from \'@ckeditor/ckeditor5-engine/src/view/text\';' ]
```